dpti/hti.py

```diff
@@ -1386,6 +1388,50 @@ def hti_phase_trans_analyze(job, jdata=None):
     return if_phase_trans
 
 
+def run_task(task_dir, machine_file, task_name, no_dp=False):
+    job_work_dir_ = glob.glob(os.path.join(task_dir, task_name + "*"))
+    assert len(job_work_dir_) == 1
+    job_work_dir = job_work_dir_[0]
+    task_dir_list = glob.glob(os.path.join(job_work_dir, "task*"))
+    task_dir_list = sorted(task_dir_list)
+    work_base_dir = os.path.join(os.getcwd())
```

Comment:
It's recommended to use pathlib for handling file paths, it's more reliable and readable than using os.path. [medium]